### ISSUE TITLE
Separate the models from the sources

### ISSUE THREAD

**Issue by DrAnaximandre** [tfunck/minc_keras#8]
If everybody is ok with that, I propose to move the sources and the models in two separate folders.
The organisation would be like:

- `src`
  - `prepare_data.py`
  - `set_images.py`
  - `anyothercode.py`
- `models`
  - `my_model_0.py`
  - `my_cnn.py`
- `run.py`

So the code would look like `python run.py --source ../data --results ../results --model my_cnn` with maybe some extra arguments for the models such as `load_weights` ?

**Comment by tfunck** [comment]
good idea. how do you get python to load a file that's defined at run time? i've never done that before, but it seems like it should be possible

**Comment by DrAnaximandre** [comment]
which file defined at runtime?
